sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -230,22 +230,53 @@ opentitan_functest(
     targets = ["cw310_rom"],
 )
 
-opentitan_functest(
-    name = "e2e_bootstrap_entry",
-    cw310 = cw310_params(
-        test_cmds = [
-            "--rom-kind=rom",
-            "--bitstream=\"$(location //hw/bitstream:rom)\"",
-            "--bootstrap=\"$(location {flash})\"",
-        ],
-    ),
-    ot_flash_binary = ":empty_test_slot_a",
-    # We don't want the `empty_test` to run, but we _also_ don't want some
-    # leftover flash image from a previous test to run.  So, bootstrap an
-    # unsigned image to force a boot failure.
-    signed = False,
-    targets = ["cw310_rom"],
-    test_harness = "//sw/host/tests/rom/e2e_bootstrap_entry",
+[otp_image(
+    name = "otp_img_e2e_bootstrap_entry_{}".format(lc_state.lower()),
+    src = "//hw/ip/otp_ctrl/data:otp_json_{}".format(lc_state.lower()),
+    overlays = STD_OTP_OVERLAYS,
```

Comment:
Looks like there's no additional overlays, so this could probably reuse an existing target like `otp_img_sigverify_key_type_*`. I'm fine with punting on that in this PR.

Reply:
This seems to be referencing only things that are already defined in `//hw/ip/otp_ctrl/data`.

Aren't these already covered by the targets `//hw/ip/otp_ctrl/data:img_${lc_state}` or did I misunderstand something?

Reply:
I think you're right, @cfrantz. We should be able to drop any of these `otp_image` targets where `overlays == STD_OTP_OVERLAYS`.

Reply:
You're right I didn't think a lot about what this test point really needs because we will do a second pass to simplify this file anyway.